internal/indexmanagement/scripts.go

```diff
@@ -427,6 +454,11 @@ function delete() {
     fi
   done
 
+  #BEE: Call deleteByQuwey here?
+  namespaceName=$NAMESPACE_NAME
+  minAgeDBQ=$MIN_AGE_DBQ
```

Comment:
You can just check if these env vars is null or not. If not null then you should be calling your defined python function here and pass these variables to that function.

Call below func inside delete():

`function deleteByQuery() {
  <-------- define variables and read them using $1, $2, ... ------------>
  python -c 'import indexManagementClient; print(indexManagementClient.deleteByQuery(<-- pass the read variables --->))'
}`